OpenDreamRuntime/Objects/Types/DreamObjectWorld.cs

```diff
@@ -64,8 +64,10 @@ public DreamObjectWorld(DreamObjectDefinition objectDefinition) :
         base(objectDefinition) {
         IoCManager.InjectDependencies(this);
 
+        SetTicklag(objectDefinition.Variables["tick_lag"]);
         SetLog(objectDefinition.Variables["log"]);
         SetFps(objectDefinition.Variables["fps"]);
```

Comment:
If a project sets `world.tick_lag` but not `world.fps` this would lead to the default fps value overriding it.